src/main/java/com/github/jnidzwetzki/bitfinex/v2/entity/currency/BitfinexCurrencyPair.java

```diff
@@ -177,9 +177,17 @@ private static String buildCacheKey(final String currency1, final String currenc
 	 */
 	private double minimumOrderSize;
 
+	/**
+	 *	If the currencypair is one of the derivatives
+	 */
+	private boolean isFuture = false;
+
 	private BitfinexCurrencyPair(final String pair1, final String pair2, final double minimumOrderSize) {
 		this.currency1 = pair1;
 		this.currency2 = pair2;
+		if (currency1.toLowerCase().contains("f") && currency2.toLowerCase().contains("f")) {
```

Comment:
This might become problematic when the two currency pairs contain a "f". 

What do you think about adding a new constructor?

`private BitfinexCurrencyPair(final String pair1, final String pair2, final double minimumOrderSize, final boolean future)`